src/web/docusaurus/docs/getting-started/environment-setup.md

```diff
@@ -288,7 +300,12 @@ Run the `pnpm db:init` script as it will populate the database with an initial s
 `db:seed` is responsible to populate the database with an initial set of data.
 :::
 
-Stop the `parser` service running. If you have Docker app installed locally make sure to stop the `parser` container before continuing to the next step.
+Docker Desktop app:
+![Image](../static/img/initialProjectDiagram.png)
+
+Docker CLI: `docker stop <container-id>` (e.g, `15649f07c6bf` in above picture)
+
+To find container-id: `docker ps -a` This will return all the docker containers, STATUS, and their PORTS
 
 Finally, restart the `parser` container by running `pnpm:services start parser`.
```

Comment:
@TueeNguyen Sorry that I wasn't clear. I meant for this line. `pnpm:services start parser` should be `pnpm services:start parser`.

Reply:
Aaah, thanks for correcting

Reply:
@Genne23v @TueeNguyen Oops, that's an unintentional typo I made in the docs. Sorry. 🙈